src/app/core/components/donate/donate.component.ts

```diff
@@ -1,264 +0,0 @@
-import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
-import { FormGroup, FormBuilder, Validators } from '@angular/forms';
-import { find } from 'lodash';
-
-import { ApiService } from '@shared/services/api/api.service';
-import { AccountService } from '@shared/services/account/account.service';
-
-import { AccountVO, BillingCardVO, BillingPaymentVO, BillingPaymentVOData } from '@models';
-
-import APP_CONFIG from '@root/app/app.config';
-import { ActivatedRoute } from '@angular/router';
-import { PromptService, PromptField } from '@shared/services/prompt/prompt.service';
-import { BillingResponse, AccountResponse } from '@shared/services/api/index.repo';
-import { MessageService } from '@shared/services/message/message.service';
-
-import { CREDIT_CARD_FIELDS, ADDRESS_FIELDS } from '@shared/components/prompt/prompt-fields';
-import { Deferred } from '@root/vendor/deferred';
-
-const DEFAULT_STORAGE_AMOUNT = 10;
-
-export enum DonationStage {
-  Storage,
-  Permanent,
-  ByteForByte,
-  Payment,
-  Complete
-}
-
-@Component({
-  selector: 'pr-donate',
-  templateUrl: './donate.component.html',
-  styleUrls: ['./donate.component.scss']
-})
-export class DonateComponent {
-  public donationStage: DonationStage = DonationStage.Storage;
-  public donationForm: FormGroup;
-  public cards: BillingCardVO[];
-  public currentAddress: any;
-  public waiting: boolean;
-
-  public storageOptions = [1, 3, 5, 10, 25];
-  public storageAmount = DEFAULT_STORAGE_AMOUNT;
-  public pricePerGb: number = APP_CONFIG.pricePerGb;
```

Comment:
https://github.com/PermanentOrg/web-app/blob/1d86f65af0d7e39801324a9c296f5e804899d68e/src/app/app.config.ts#L3

Looks like there's one more reference to `APP_CONFIG.pricePerGb` in `PhaseProgressComponent`: https://github.com/PermanentOrg/web-app/blob/1d86f65af0d7e39801324a9c296f5e804899d68e/src/app/pledge/components/phase-progress/phase-progress.component.ts#L74

Which is maybe a thread to tug a bit harder on? But maybe in a separate PR?